velox/functions/prestosql/aggregates/SimpleNumericAggregate.h

```diff
@@ -71,6 +71,7 @@ class SimpleNumericAggregate : public exec::Aggregate {
   template <
       bool tableHasNulls,
       typename TData = TResult,
+      typename TValue = TInput,
```

Comment:
Would you update the comment. It looks like it is no longer accurate / complete. Template parameter names are now confusing. Data, Value, Input and Result. Hard-to-tell which is which.